build/yamls/antrea-octant.yml

```diff
@@ -0,0 +1,72 @@
+# yaml template for octant and antrea-octant-plugin
+# Right config parameters and Docker image must be specified.
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: antrea-octant
+  namespace: kube-system
+  labels:
+    app: antrea
+    component: antrea-octant
+spec:
+  replicas: 1
+  strategy:
+    # Ensure the existing Pod is killed before the new one is created.
+    type: Recreate
+  selector:
+    matchLabels:
+      app: antrea
+      component: antrea-octant
+  template:
+    metadata:
+      labels:
+        app: antrea
+        component: antrea-octant
+    spec:
+      nodeSelector:
+        # Add the role to support running antrea-octant on specific node
+        # In this way, users can see UI via a fixed URL as http://$HOSTNAME:8900
+        role: antrea-octant
+        # Note: beta.kubernetes.io/os is targeted for removal in K8s v1.18, if running Antrea with 1.18
+        # or higher uncomment the following line and remove beta.kubernetes.io/os.
+        #kubernetes.io/os: linux
+        beta.kubernetes.io/os: linux
+      hostNetwork: true
```

Comment:
Thanks for pointing out. I have changed this part in the next commit. I think hostnetwork is a must if we want to see octant via URL with hostIP/hostName. According to my deployment on the live testbed, octant start-up env like OCTANT_ACCEPTED_HOSTS and OCTANT_LISTENER_ADDR only work when I mark hostNetwork to be true, otherwise, even though octant is running fine in the pod, we can not access it via HostIP.
But linux Node may not be necessary and I have removed it in the next commit.

Reply:
Thanks for pointing out. Port mapping can also work. Previously in my set up, certain port on the host is not open and it actually works after I add an ACCEPT rule to iptables.

Reply:
I change this part to port mapping since hostNetwork is only supported for Linux. Please help check the latest version.